### PR TITLE
Add qscout_tutorial.ipynb to qiskit-superstaq/examples

### PR DESCRIPTION
**Pull request by hallben4**

### CODE REVIEW

**Review by singular-value — CHANGES_REQUESTED** [review]
Thanks. A few requests:
(1) Please remove the cirq examples/imports from the notebook. (And correspondingly, remove the qiskit examples/imports from the css notebook)
(2) Since there is already a qscout_compile.ipynb notebook, I think it would make sense to combine those two. (Ditto in css)
(3) Looks like the notebook is failing tests. Please check out the output and fix.

**Comment by pnoell** [comment]
Will add MPL drawer to example requirements and remove #!pip install in notebook

File: examples/qscout_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "395bf672",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "token = \"1f59b6240f2b6c9b007b4cfe41b8f468713b95dedcf9b796a825edcf64f54e53\"\n",
```
Comment (richrines1) [inline examples/qscout_tutorial.ipynb:68]: oops - need to change this token

**Comment by hallben4** [comment]
The notebook keeps failing because the it can't find the superstaq token. But we're not supposed to put the token on github. How do we resolve this?

File: examples/qscout_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m23.0\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m23.0.1\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "! pip install qiskit-superstaq\n",
+    "! pip install typing-extensions\n",
+    "! pip install pylatexenc"
```
Comment (richrines1) [inline examples/qscout_tutorial.ipynb:138]: as per the box above, these should be commented:
```suggestion
    "# ! pip install qiskit-superstaq\n",
    "# Required for pretty-printing circuit diagrams:\n",
    "# ! pip install pylatexenc"
```
(also removed typing-extensions, as it's already required by gss)

the notebook should also be re-run to remove the output block

File: examples/qscout_tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,732 @@
...
+   "id": "6b6be06f",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'provider' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
```
Comment (richrines1) [inline examples/qscout_tutorial.ipynb:657]: re-run or fix to get rid of this error

File: examples/qscout_tutorial.ipynb [hunk 4]
```diff
@@ -0,0 +1,732 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "395bf672",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# token = \"YOUR_TOKEN_HERE\"\n",
+    "# os.environ[\"SUPERSTAQ_API_KEY\"] = token"
```
Comment (richrines1) [inline examples/qscout_tutorial.ipynb:170]: maybe add a note saying to either uncomment these and add a token, or set the SUPERSTAQ_API_KEY variable in your local env

**Comment by richrines1** [comment]
@hallben4 do you mind reopening this in https://github.com/SupertechLabs/client-superstaq/ (reach out if you need help)

**Pull request closed without merging**